app/controllers/application_controller.rb

```diff
@@ -22,4 +27,9 @@ def configure_permitted_parameters
     update_attrs = %i[password password_confirmation current_password]
     devise_parameter_sanitizer.permit :account_update, keys: update_attrs
   end
+
+  def reload_yaml
+    [QuestionnaireData, SummativeQuestionnaire ].each { |m| m.reload(true) }
+  end
+
 end
```

Comment:
I wasn't expecting to see a separate SummativeQuestionnaire object - a questionnaire is a questionnaire, whether it's being shown in the formative stage or summative stage - it's just how it's graded and answers recorded.